server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -283,6 +295,19 @@ export class MongoCollection<T> implements core.ICollection<T>, core.IRetryable
 
         return properties;
     }
+
+    private sanitizeError(error: any) {
+        if (error) {
+            try {
+                redactJsonKeys(error);
```

Comment:
Interesting, does this apply the changes to the error object itself, without the need of returning a new copy with redacted fields?

Reply:
It depends on the `serialize` option specified along with the paths above. If it is set to false, then the original object will be mutated (have a comment describing this).

Reply:
Got it! Thanks for the details, and apologies for missing the comment earlier. Makes sense :)

Reply:
no worries :)